test/image/mocks/bar_attrs_relative.json

```diff
@@ -2,27 +2,37 @@
     "data":[
         {
             "width":[1,0.8,0.6,0.4],
+            "text":[1,2,3333333333,4],
+            "textposition":"outside",
             "y":[1,2,3,4],
             "x":[1,2,3,4],
             "type":"bar"
         }, {
             "width":[0.4,0.6,0.8,1],
+            "text":["Three",2,"outside text",0],
```

Comment:
@alexcjohnson OK, I'll change `auto`:
- if it fits inside, the label will be drawn inside
- if it doesn't, the label will be drawn wherever is larger